app.json

```diff
@@ -2,9 +2,20 @@
   "name": "api.mozillascience.org",
   "description": "API server for science.mozilla.org",
   "addons": [
-    "heroku-postgresql"
+    "heroku-postgresql:hobby-dev"
   ],
   "scripts": {
-    "postdeploy": "python app/manage.py migrate"
+    "postdeploy": "sh scripts/postdeploy.sh"
+  },
+  "env": {
+    "DJANGO_ENV": "review",
```

Comment:
@arush0311 it's used for every build that heroku runs for this app. So it should be used for staging/production.

Having this env is fine as long as you override it in staging/prod